lib/chai/core/assertions.js

```diff
@@ -2116,6 +2110,13 @@ function assertOwnPropertyDescriptor (name, descriptor, msg) {
 Assertion.addMethod('ownPropertyDescriptor', assertOwnPropertyDescriptor);
 Assertion.addMethod('haveOwnPropertyDescriptor', assertOwnPropertyDescriptor);
 
+/**
+ *
```

Comment:
maybe add a description? "Flags that the `length` property should be asserted in this chain"